example01/src/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@
 
```

Comment:
This is a nice example, but I would prefer to add a different example to highlight this behaviour and nothing else.
This diverts the original example and makes any previous references to it in the docs invalid.

Reply:
I have reverted example01 to the original version